File: Auto-generated-tasks/Difficulty-2/Task-5/Solution.pas
Var
	Cnt:=7, i1, Itog:=1:integer;
	T:Array[0..Cnt] of real;
Begin
	for i1:= 0 to Cnt do begin
		if (T[i1] > T[Itog]) then begin
			Itog := i1;
		end;
	end;
	WriteLn("Itog=", Itog);
End.

File: Auto-generated-tasks/Difficulty-2/Task-5/Solution.php
<?
$Cnt = 7;
$T = array_fill(0, $Cnt, 0.0);
$i1=0;
$Itog = 1;
for ($i1 = 0; $i1 < $Cnt; $i1++) {
	if ($T[$i1] > $T[$Itog])
		$Itog = $i1;
}
print("Itog=" + $Itog);
?>